### PR TITLE
GoogleSheetsConnector demo 

### PR DESCRIPTION
**Pull request by bre1470**

### CODE REVIEW

**Review by pawelfus — APPROVED** [review]
Thanks!

File: samples/highcharts/data-tools/googlesheets-html/demo.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/modules/data-tools.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<p>
+    Use the fields below to configure the GoogleSheetsConnector. It will load
```
Comment (karolkolodziej) [inline samples/highcharts/data-tools/googlesheets-html/demo.html:6]: Is it ok to share this sheet so it's easily accessible and we see how it is structured?
https://docs.google.com/spreadsheets/d/1U17c4GljMWpgk1bcTvUzIuWT8vdOnlCBHTm5S8Jh8tw/edit#gid=0

**Review by karolkolodziej — APPROVED** [review]

File: samples/highcharts/data-tools/googlesheets-html/demo.html (hunk 1 above)
Comment (bre1470) [reply samples/highcharts/data-tools/googlesheets-html/demo.html:6]: We can't do this, but it has a form, so you can test with your own Google Spreadsheets.

**Review by pawellysy — APPROVED** [review]
LGTM!

**Review by goransle — APPROVED** [review]

File: samples/highcharts/data-tools/googlesheets-html/demo.css [hunk 2]
```diff
@@ -0,0 +1,41 @@
+:root {
+    --color-background: #fff;
+    --color-foreground: #000;
```
Comment (sebastianbochan) [inline samples/highcharts/data-tools/googlesheets-html/demo.css:3]: As I see it is used by the `color` param which means text. I suggest to use a name like `--color-text` or something similar which is more clear.

**Review by sebastianbochan — CHANGES_REQUESTED** [review]

**Review by sebastianbochan — APPROVED** [review]

**Pull request merged**